demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -296,13 +296,13 @@ def run_validation_on_all_packs(self):
             all_packs_valid.add(self.conf_json_validator.is_valid_conf_json())
 
         count = 1
-        all_packs = os.listdir(PACKS_DIR) if os.listdir(PACKS_DIR) else []
+        # Ignores local files (non-packs), such as .DS_STORE on MacOS
+        all_packs = list(filter(os.path.isdir, [os.path.join(PACKS_DIR, p) for p in os.listdir(PACKS_DIR)]))
```

Comment:
I find list comprehension a little easier to read, but in this case it isn't a big deal, leaving this suggestion for you to decide
```suggestion
        all_packs = [os.path.join(PACKS_DIR, pack) for pack in os.listdir(PACKS_DIR) if os.path.isdir(pack)]
```

Reply:
here it'd be 
```
 all_packs = [os.path.join(PACKS_DIR, pack) for pack in os.listdir(PACKS_DIR) if os.path.isdir(os.path.join(PACKS_DIR, pack))]
```
which isn't elegant imo, so i'll leave it with`filter`